OpenRCT2.xcodeproj/project.pbxproj

```diff
@@ -2155,6 +2162,8 @@
 					rct2_data,
 					rw,
 					rw,
+					"-lssl",
```

Comment:
All the other libs used for linking are listed elsewhere, please follow their convention